### PR TITLE
Update SQL BBEs for V2

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

### CODE REVIEW

File: examples/mysql-atomic-batch-execute-operation/initialize.bal [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            registrationID INTEGER UNIQUE, creditLimit DOUBLE, country  VARCHAR(300),
+            PRIMARY KEY (customerId))`);
+
+    // Adds records to the newly-created table.
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (firstName, lastName, registrationID,creditLimit,country) VALUES
+             ('Peter', 'Stuart', 1, 5000.75, 'USA')`);
+
+    check mysqlClient.close();
+}
```
Comment (daneshk) [inline examples/mysql-atomic-batch-execute-operation/initialize.bal:23]:
```suggestion
}

```

File: examples/mysql-execute-operation/cleanup.bal [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+// Cleans up the database after running the example.
+function cleanup() returns sql:Error? {
+    mysql:Client mysqlClient = check new (user = "root", password = "Test@123");
+
+    // Cleans the database.
+    _ = check mysqlClient->execute(`DROP DATABASE CUSTOMER`);
+
+    // Closes the MySQL client.
+    check mysqlClient.close();
+}
```
Comment (daneshk) [inline examples/mysql-execute-operation/cleanup.bal:13]:
```suggestion
}

```

File: examples/mysql-execute-operation/initialize.bal [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    // and `lastInsertedId` with the auto-generated ID of the last row.
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (firstName, lastName, registrationID,creditLimit,country) VALUES
+            ('Peter','Stuart', 1, 5000.75, 'USA')`);
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (firstName, lastName, registrationID,creditLimit,country) VALUES
+            ('Dan', 'Brown', 2, 10000, 'UK')`);
+
+    check mysqlClient.close();
+}
```
Comment (daneshk) [inline examples/mysql-execute-operation/initialize.bal:28]:
```suggestion
}

```

File: examples/mysql-query-column-mapping/initialize.bal [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            PRIMARY KEY (customer_id))`);
+
+    // Adds the records to the newly-created table.
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (first_name, last_name) VALUES ('Peter','Stuart')`);
+    _ = check mysqlClient->execute(`INSERT INTO CUSTOMER.Customers
+            (first_name, last_name) VALUES ('Dan', 'Brown')`);
+
+    check mysqlClient.close();
+}
```
Comment (daneshk) [inline examples/mysql-query-column-mapping/initialize.bal:24]:
```suggestion
}

```

**Review by daneshk — APPROVED** [review]
Add minor comments. Overall LGTM

File: examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md [hunk 5]
```diff
@@ -1,13 +1,19 @@
 # Atomic batch execute
 
-This BBE demonstrates how to use the MySQL client to execute a batch of
-DDL/DML operations with the help of a `transaction` to achieve the atomic behaviour.
-Note that the MySQL database driver JAR should be defined in
+This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations with the help of a 
+`transaction` to achieve the atomic behaviour. Note that the MySQL database driver JAR should be defined in
```
Comment (praneesha) [inline examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md:4]:
```suggestion
`transaction` to achieve the atomic behaviour. Note that the MySQL database driver JAR should be defined in
```
Let's separate section for the note in Markdown format.

File: examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md [hunk 6]
```diff
@@ -1,13 +1,19 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_atomic_batch_execute_operation.bal :::
 
-::: out mysql_atomic_batch_execute_operation.out :::
\ No newline at end of file
+::: out mysql_atomic_batch_execute_operation.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md:15]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-batch-execute-operation/mysql_batch_execute_operation.md [hunk 7]
```diff
@@ -3,10 +3,17 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_batch_execute_operation.bal :::
 
-::: out mysql_batch_execute_operation.out :::
\ No newline at end of file
+::: out mysql_batch_execute_operation.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-batch-execute-operation/mysql_batch_execute_operation.md:15]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-call-stored-procedures/initialize.bal [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    _ = check mysqlClient->execute(`CREATE TABLE STUDENT.Student
+            (id INT AUTO_INCREMENT, age INT, name VARCHAR(255),
+            PRIMARY KEY (id))`);
+
+    // Creates the necessary stored procedures using the execute command.
+    _ = check mysqlClient->execute(`CREATE PROCEDURE STUDENT.GetCount
+        (INOUT pID INT, OUT totalCount INT) BEGIN SELECT age INTO pID FROM
+        Student WHERE id = pID; SELECT COUNT(*) INTO totalCount FROM Student;
+        END`);
+    // Inserts necessary data.
```
Comment (praneesha) [inline examples/mysql-call-stored-procedures/initialize.bal:21]:
```suggestion
    // Inserts the necessary data.
```

File: examples/mysql-call-stored-procedures/mysql_call_stored_procedures.md [hunk 9]
```diff
@@ -3,10 +3,17 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_call_stored_procedures.bal :::
 
-::: out mysql_call_stored_procedures.out :::
\ No newline at end of file
+::: out mysql_call_stored_procedures.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-call-stored-procedures/mysql_call_stored_procedures.md:15]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-execute-operation/mysql_execute_operation.md [hunk 10]
```diff
@@ -2,10 +2,17 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_execute_operation.bal :::
 
-::: out mysql_execute_operation.out :::
\ No newline at end of file
+::: out mysql_execute_operation.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-execute-operation/mysql_execute_operation.md:14]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-query-column-mapping/mysql_query_column_mapping.md [hunk 11]
```diff
@@ -2,10 +2,17 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_query_column_mapping.bal :::
 
-::: out mysql_query_column_mapping.out :::
\ No newline at end of file
+::: out mysql_query_column_mapping.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-query-column-mapping/mysql_query_column_mapping.md:14]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-query-operation/mysql_query_operation.md [hunk 12]
```diff
@@ -3,10 +3,17 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_query_operation.bal :::
 
-::: out mysql_query_operation.out :::
\ No newline at end of file
+::: out mysql_query_operation.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-query-operation/mysql_query_operation.md:15]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-query-row-operation/mysql_query_row_operation.md [hunk 13]
```diff
@@ -1,12 +1,18 @@
...
-recommended to use a MySQL driver version greater than 8.0.13.<br><br>
+recommended to use a MySQL driver version greater than 8.0.13.
 
 ::: code mysql_query_row_operation.bal :::
 
-::: out mysql_query_row_operation.out :::
\ No newline at end of file
+::: out mysql_query_row_operation.out :::
+
+The following util files will initialize the test database before running the bbe and cleans it up afterwards.
```
Comment (praneesha) [inline examples/mysql-query-row-operation/mysql_query_row_operation.md:14]:
```suggestion
The following util files will initialize the test database before running the BBE and clean it up afterward.
```

File: examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.bal [hunk 14]
```diff
@@ -7,21 +7,18 @@ public function main() returns error? {
...
     // The transaction block can be used to roll back if any error occurred.
     transaction {
-        _ = check mysqlClient->execute(`INSERT INTO Customers (firstName, 
-                     lastName, registrationID, creditLimit, country) VALUES 
-                     ('Linda', 'Jones', 4, 10000.75, 'USA')`);
+        _ = check mysqlClient->execute(`INSERT INTO Customers (firstName, lastName, registrationID, creditLimit,
+                                        country) VALUES ('Linda', 'Jones', 4, 10000.75, 'USA')`);
         log:printInfo("First query executed successfully.");
 
         // Insert Customer record which violates the unique
```
Comment (praneesha) [inline examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.bal:18]:
```suggestion
        // Insert a `customer` record, which violates the unique
```

File: examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md [hunk 15]
```diff
@@ -1,13 +1,19 @@
 # Atomic batch execute
 
-This BBE demonstrates how to use the MySQL client to execute a batch of
-DDL/DML operations with the help of a `transaction` to achieve the atomic behaviour.
-Note that the MySQL database driver JAR should be defined in
+This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations with the help of a 
+`transaction` to achieve the atomic behaviour. Note that the MySQL database driver JAR should be defined in
 the `Ballerina.toml` file as a dependency.
+
 For a sample configuration and more information on the underlying module, see the [MySQL module](https://docs.central.ballerina.io/ballerinax/mysql/latest/).
```
Comment (praneesha) [inline examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.md:7]:
```suggestion
For a sample configuration and more information on the underlying module, see the [`mysql` module](https://docs.central.ballerina.io/ballerinax/mysql/latest/).
```
Please check and fix all the occurrences.

File: examples/mysql-atomic-batch-execute-operation/initialize.bal (hunk 1 above)
Comment (niveathika) [reply examples/mysql-atomic-batch-execute-operation/initialize.bal]: Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)

File: examples/mysql-execute-operation/cleanup.bal (hunk 2 above)
Comment (niveathika) [reply examples/mysql-execute-operation/cleanup.bal]: Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)

File: examples/mysql-execute-operation/initialize.bal (hunk 3 above)
Comment (niveathika) [reply examples/mysql-execute-operation/initialize.bal]: Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)

File: examples/mysql-query-column-mapping/initialize.bal (hunk 4 above)
Comment (niveathika) [reply examples/mysql-query-column-mapping/initialize.bal]: Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)

**Review by praneesha — APPROVED** [review]

**Pull request merged**